docs/PIPELINES-CONFIG.md

```diff
@@ -0,0 +1,269 @@
+    name: cosign-pub
+    namespace: $APP_NAMESPACE
+type: Opaque
+EOF
+```
+
+#### Step 5: Setting up gitops authentication secret under deployment namespaces
+
+If you are using GitLab, you will need to create the GitLab authentication secret with your GitLab PAT as follows:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret generic "gitlab-auth-secret" \
+    --from-literal=password=<gitlab_pat> \
+    --from-literal=username=oauth2 \
+    --type=kubernetes.io/basic-auth \
+    --dry-run=client -o yaml | kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+Otherwise, you will need to create a Git authentication secret with your PAT (e.g. GitHub PAT) as follows:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret generic "gitops-auth-secret" \
+    --from-literal=password=<git_pat> \
+    --type=kubernetes.io/basic-auth \
+    --dry-run=client -o yaml | kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+#### Step 6: Setting up pipelines secret under deployment namespaces
+
+You will need to create a pipeline secret containing the webhook secret for the Git organization for deployments to have proper access via the tekton pipelines:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret generic "pipelines-secret" \
+    --from-literal=webhook.secret=<webhook_secret> \
+    --dry-run=client -o yaml | kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+#### Step 7: Setting up quay image registry secret under deployment namespaces
+
+For accessing the quay image registry, you'll need to create a secret
+to store the docker config json file:
+
+```sh
+kubectl -n $APP_NAMESPACE create secret docker-registry "rhdh-image-registry-token" \
+    --from-file=.dockerconfigjson="<dockerconfig_jsonfile>" --dry-run=client -o yaml | \
+    kubectl -n $APP_NAMESPACE apply --filename - --overwrite=true >/dev/null
+```
+
+Then for both the `default` and `pipeline` service accounts under the deployment namespace, you will need to tie in the image registry secret you just created:
+
+```sh
+kubectl -n $APP_NAMESPACE patch serviceaccounts "<default|pipeline>" --patch "
+secrets:
+- name: rhdh-image-registry-token
+imagePullSecrets:
+- name: rhdh-image-registry-token
+" >/dev/null
+```
+
+#### Step 8: Updating Plugins
+
+To include the [Tekton plugins list](../dynamic-plugins/tekton-plugins.yaml) we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
+
+![Dynamic Plugins Example](../assets/dynamic-plugins-example.png)
+
+Edit the associated `yaml` file to include the contents of the [Tekton plugins list](../dynamic-plugins/tekton-plugins.yaml) under the `plugins` section:
+
+![Dynamic Plugins Example 3](../assets/dynamic-plugins-example-3.png)
+
+Alternatively, we can use this series of commands to perform the same task with `kubectl` and `yq` using the [`tekton-plugins.yaml`](../dynamic-plugins/tekton-plugins.yaml):
```

Comment:
I think having sub-section might help here too

Reply:
Done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/143b26952fac4d4be9bb4d0942fab099b69d7dc5